tests/BuildSystem/Build/failing-commands.llbuild

```diff
@@ -5,7 +5,7 @@
 # RUN: rm -rf %t.build
 # RUN: mkdir -p %t.build
 # RUN: cp %s %t.build/build.llbuild
-# RUN: %{llbuild} buildsystem build --serial --chdir %t.build > %t.out || true
+# RUN: %{llbuild} buildsystem build --serial --chdir %t.build &> %t.out || true
```

Comment:
I don't think we want this change either, I think it is a feature not a bug that the subcommands stderr is funneled into the build commands stdout (at least for now).